packages/core/useStepper/index.ts

```diff
@@ -0,0 +1,83 @@
+import { computed, ref } from 'vue-demi'
+
+export function useStepper<T extends string>(steps: readonly T[], initial?: T) {
```

Comment:
```suggestion
export function useStepper<T>(steps: readonly T[], initial?: T) {
```

Can we have it arbitrary? I might want to use objects.

Reply:
Yes I don't see a reason why it should only be strings. I use that mostly but you're right that objects can be useful. Also added basic tests for that.